hydra-node/src/Hydra/API/HTTPServer.hs

```diff
@@ -51,6 +51,7 @@ instance Arbitrary tx => Arbitrary (DraftCommitTxResponse tx) where
 data DraftCommitTxRequest tx
   = SimpleCommitRequest
       { utxoToCommit :: UTxOType tx
+      , amount :: Maybe Coin
```

Comment:
why is this change not reflected in hydra-node/json-schemas/api.yaml?

Reply:
We kept the json instance backwards compatible but you are right we should alter the api.yaml as well.